slam-libraries/viam-orb-slam3/orbslam_server_v1_test.cc

```diff
@@ -100,6 +104,7 @@ BOOST_AUTO_TEST_CASE(ParseAndValidateArguments_valid_config) {
     BOOST_TEST(slamService.map_rate_sec.count() == chrono::seconds(60).count());
     BOOST_TEST(slamService.camera_name == "color");
     BOOST_TEST(slamService.offlineFlag == false);
+    BOOST_TEST(slamService.delete_processed_data == false);
 }
 
 BOOST_AUTO_TEST_CASE(
```

Comment:
For this test case, I would include the delete_processed_data parameter, since it should always be sent by rdk.